src/test/java/kg/jarkyn/server/ReceivingSocketTest.java

```diff
@@ -0,0 +1,62 @@
+package kg.jarkyn.server;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+import static org.junit.Assert.assertTrue;
+
+public class ReceivingSocketTest {
```

Comment:
How much value does this test class actually add?
I'd avoid opening and using network connections in tests as much as possible.